<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDecision</class>
 <widget class="QDialog" name="frmDecision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terra.qrc">
    <normaloff>:/Icons/PNG/terra.png</normaloff>:/Icons/PNG/terra.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="linea1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linea2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linea3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExtra">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="terra.qrc">:/Icons/PNG/interrogantes.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="terra.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>frmDecision</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmDecision</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExtra</sender>
   <signal>clicked()</signal>
   <receiver>frmDecision</receiver>
   <slot>Anular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Anular()</slot>
 </slots>
</ui>
